### PR TITLE
initialize setup for dashboard

### PR DESCRIPTION
**Pull request by kirbygit**
- initialize dashboard
- two modals for My Games & Open Games
- to do:
- will try to implement this feature on dashboard once everything is done
screen shot 2017-07-05 at 3 23 36 pm
- add some rules on how to play chess
- will show user name and a simple default avatar to the page

### CODE REVIEW

**Comment by kirbygit** [comment]
@nnadkarny lmk when we get a chance to pair program

**Comment by nnadkarny** [comment]
@kirbygit looks good, I can see the games listed in the "Open Games" modal.  Adding games to the "My Games" modal should be pretty straightforward as well.

Maybe we can put a user info card above the "My Games" and "Open Games" modals?  Something with nickname, email address, etc.  We could wrap it in an if statement:  if registered user, show the card, else show a sign-up button?

@jwmunn, @fistoriza, @Phatkid98, checkout this branch when you get a chance and let us know what else you think should go in the dashboard.  Does it make sense to include placeholders for stuff we want to add later (like games won/lost/drawn)?

**Comment by fistoriza** [comment]
@nnadkarny @kirbygit @jwmunn @Phatkid98 yes i think it's good to have placeholders for items we want to add later (player's record - won/lost/drawn, profile pic, badges, etc). 

about the modals, the Open Games will be for the games that are still on going right? but what will be listed on My Games?

File: app/views/dashboards/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- My Games modal -->
```
Comment (jwmunn) [inline app/views/dashboards/index.html.erb]: Change this comment and the one below to `<!-- Open Games Modal -->` to avoid confusion.

**Comment by jwmunn** [comment]
+1 @fistoriza What will be listed in *My Games*? Open Games works well, nice job. The `modal-lg` class could be added to the `modal-dialog` div so it's wider and  the join buttons have a smaller chance of wrapping to the next line.

**Pull request merged**